src/Common/src/Interop/Windows/sspicli/SecuritySafeHandles.cs

```diff
@@ -838,41 +780,27 @@ public new IntPtr DangerousGetHandle()
             ref Interop.SspiCli.ContextFlags outFlags)
         {
 #if TRACE_VERBOSE
```

Comment:
Is there any reason to keep TRACE_VERBOSE now, assuming it's all protected by an IsEnabled test?